.github/workflows/check-external-contributor.yml

```diff
@@ -0,0 +1,16 @@
+name: External Fork check
+on:
+  pull_request: {}
+  
+jobs:
+  check-external:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR and label
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```

The token is already available as `GITHUB_TOKEN`, no need to set it again as env var of different name